src/status_im/chat/handlers.cljs

```diff
@@ -172,14 +172,15 @@
                        :added-to-at timestamp
                        :timestamp   timestamp
                        :is-active   true}
-            exists?   (chats/exists? group-id)]
-        (when (or (not exists?) (chats/new-update? timestamp group-id))
+            exists?   (not (nil? chat))]
```

Comment:
Technically, `(nil? chat)` is not necessary and `(not chat)` will work just as good, but minor point.

Reply:
@janherich ok, i'll get rid of `exists?` binding at all, because `(if chat)` reads as clear as `(if exists?)`